### ISSUE TITLE
🦠 Model Request: MolE molecular embeddings

### ISSUE THREAD

**Issue by miquelduranfrigola** [ersilia-os/ersilia#1385]
### Model Name

MolE molecular embeddings

### Model Description

MolE is a foundation model for chemistry developed by Recursion. It combines geometric deep learning with transformers, to learn a meaningful representation of molecules. MolE leverages extensive labeled and unlabeled datasets in two pretraining steps. First it follows a novel self-supervised strategy using the graph representation of ~842 million molecules designed to properly learn to represent chemical structures. It is followed by a massive multi-task training to assimilate biological information.

### Slug

mole-embeddings

### Tag

Embedding

### Publication

https://www.nature.com/articles/s41467-024-53751-y

### Source Code

https://github.com/recursionpharma/mole_public

### License

MIT

**Comment by miquelduranfrigola** [comment]
Unfortunately I cannot incorporate this model yet since the pre-trained models are not yet available in the repository. I have opened an issue: https://github.com/recursionpharma/mole_public/issues/2

**Comment by miquelduranfrigola** [comment]
Update: I got an answer to my previous query and, disappointingly, they will not release the weights of the model even when they said they would do so in the repo

**Comment by GemmaTuron** [comment]
So, should we close this model and eliminate the repo @miquelduranfrigola ? Disappointing indeed

**Comment by miquelduranfrigola** [comment]
Yes I think we close it. Give me the green light and I will do it

**Comment by GemmaTuron** [comment]
They released the version pretrained in GuacaMol and Chembl: https://codeocean.com/capsule/2105466/tree/v1
Do we still want to include those or not @miquelduranfrigola ? 

Maybe this can also be helpful. In this [paper](https://www.nature.com/articles/s41467-025-58804-4#data-availability) they pretrain their own MolE if I understood correctly at a quick glance? 
Code: https://github.com/rolayoalarcon/MolE

**Comment by arnaucoma24** [comment]
Hey! 

Just came across this issue when creating a new Model Request for the MolE model @GemmaTuron just mentioned (MolE - Molecular representation through redundancy reduced Embeddings from this [paper](https://www.nature.com/articles/s41467-025-58804-4)).

Apparently they use the same name, which can be a bit misleading - at least it was for me. Let's keep things separate.

**Comment by arnaucoma24** [comment]
Yes, I already did :)

https://github.com/ersilia-os/ersilia/issues/1615

**Comment by GemmaTuron** [comment]
@miquelduranfrigola I have deleted all repos associated with this model request.
Do we want to attempt the incorporation from the pretrained model trained on Guacamol / Chembl as I indicate above or we want to close this issue?

**Comment by miquelduranfrigola** [comment]
Please see this error: https://github.com/ersilia-os/ersilia-stats/issues/110
I don't know why the entry was not successfully created in AirTable.

**Comment by GemmaTuron** [comment]
good thanks! too many MolE whatever names.. we will have to figure out better slugs, maybe not molE-XGBoost but mole-amr or something similar? becaus emole-xgboost is only pointing at the method but not the actual use of the model

**Comment by Marina18** [comment]
We should incorporate this model @arnaucoma24 please up this in your task list

**Comment by arnaucoma24** [comment]
Hi @Marina18 — picking this up now, starting work on finishing the eos1tt2 (MolE molecular embeddings) incorporation. Will keep this thread updated as it progresses.

**Comment by arnaucoma24** [comment]
`eos1tt2` (MolE molecular embeddings) incorporated in [PR #2](https://github.com/ersilia-os/eos1tt2/pull/2), verified equivalent to the original code — closing (distinct from `eos4ex3`/`eos6m2k`).
